Source/Core/InputCommon/ControllerEmu/ControlGroup/ModifySettingsButton.h

```diff
@@ -18,10 +18,11 @@ class ModifySettingsButton : public Buttons
 
   void AddInput(std::string button_name, bool toggle = false);
 
+  // Modifies the state
   void GetState();
 
-  const std::vector<bool>& isSettingToggled() const;
-  const std::vector<bool>& getSettingsModifier() const;
+  const std::vector<bool>& IsSettingToggled() const;
+  const std::vector<bool>& GetSettingsModifier() const;
 
 private:
   std::vector<bool> threshold_exceeded;  // internal calculation (if "state" was above threshold)
```

Comment:
You renamed members in the `ExpressionParser` to have the `m_` prefix, but didn't touch those. Is this an oversight, or something you intend to do in another PR?

Reply:
It's not like I specifically looked for anything that didn't have m_, I just fixed the naming of things I ended up directly working with. I don't see the problem here.